src/test/java/ru/training/at/hw1/mainTestClasses/AdditionTest.java

```diff
@@ -0,0 +1,28 @@
+package ru.training.at.hw1.mainTestClasses;
+
+import com.epam.tat.module4.Calculator;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import ru.training.at.hw1.DataProviders;
+
+public class AdditionTest {
+
+    @Test(dataProviderClass = DataProviders.class, dataProvider = "addition data - long",
+          groups = "additionAndSubtraction")
+    public void additionTestingLong(long firstNum, long secondNum, long expectedResult) {
+        Calculator calculator = new Calculator();
```

Comment:
Looks like SystemUnderTest-initialization, please move it to the appropriate place. Pay attention that all Test Classes have this point in common